burger-app/src/components/Main/Prices/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+export class Prices extends Component {
```

Comment:
You do not need to have `Prices` component as `class`. Refactor to be a simply function that returns JSX code